src/screens/Game/GamePage/index.css

```diff
@@ -1,33 +1,30 @@
 .gameConfigContainer {
-    background: var(--background-transparent);
+    background: var(--background-darker);
     box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-    margin: auto;
     display: flex;
-    grid-area: content;
     height: 100vh;
-    width: var(--content-width);
-    justify-content: center;
+    justify-content: stretch;
+    position: relative;
 }
 
 .gameConfig {
     display: flex;
     align-self: center;
     max-width: 1000px;
-    margin: 44px 0 44px 44px;
 }
 
-.gametitle {
+.gameConfigContainer .title {
     font-weight: 500;
     font-size: 25px;
     line-height: 30px;
     color: #f79707;
+    text-align: left;
 }
 
-.gameImg {
-    width: 210px;
-    height: 280px;
-    border-radius: 0.5rem;
-    object-fit: cover;
+.gamePicture {
+    flex-basis: 40%;
+    flex-shrink: 0;
+    flex-grow: 0;
```

Comment:
I tried to use the whorthand `flex: 40% 0 0` but the browser is treating that as `40% 1 0` for some reason so I had to split the properties